src/mrack/providers/openstack.py

```diff
@@ -278,6 +278,25 @@ def validate_host(self, req):
 
         return True
 
+    async def prepare_provisioning(self, reqs):
+        """
+        Prepare provisioning.
+
+        Load missing images if they are not in provisioning-config.yaml
+        """
+        prepare_images = list(
+            set([req["image"] for req in reqs if req["image"] not in self.images])
+        )
+        if prepare_images:
+            if len(prepare_images) > 1:
```

Comment:
Why this if? IMO it should work even without it:
```
>>> abc = ["image"]
>>> ", ".join(abc)
'image'
>>> foo = ["image", "another"]
>>> ", ".join(foo)
'image, another'
```
so simple `im_list = ", ".join(prepare_images)` should return the expected even with one value.

Reply:
and also with 0 elements :) , though it is not our case :
```
>>> ", ".join([])
''
```

Reply:
my bad, you're right

Reply:
let me do a patch for this not needed code

Reply:
https://github.com/neoave/mrack/pull/66/commits/ca82c4604a2e9eefdd2ebbc1884b07384e567f5a